src/users/containers/utils/user-roles-util.ts

```diff
@@ -1,11 +1,11 @@
-import { AppConstants } from '../../../app/app.constants';
 import {propsExist} from '../../../../api/lib/objectUtilities';
```

Comment:
Can we please try avoiding a cross import between api and frontend? We've had issues with this in the past

Reply:
Will do

Reply:
Done